src/main/java/org/apache/commons/lang3/EnumUtils.java

```diff
@@ -119,13 +119,29 @@ public EnumUtils() {
      * @return the enum, null if not found
      */
     public static <E extends Enum<E>> E getEnum(final Class<E> enumClass, final String enumName) {
+        return getEnum(enumClass, enumName, null);
+    }
+
+    /**
+     * <p>Gets the enum for the class, returning {@code defaultEnum} if not found.</p>
+     *
+     * <p>This method differs from {@link Enum#valueOf} in that it does not throw an exception
+     * for an invalid enum name.</p>
+     *
+     * @param <E> the type of the enumeration
+     * @param enumClass   the class of the enum to query, not null
+     * @param enumName    the enum name, null returns default enum
+     * @param defaultEnum the default enum
+     * @return the enum, default enum if not found
```

Comment:
Provide a `@since` tag for the new method.

Reply:
@Stzx, may I know what version number should I put ?

Reply:
`3.10` :)

Reply:
@Stzx, Thanks for your guidance.